### DISCUSSION TITLE
Meeting settings - internal

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#127] [Ideas]
Hi
All meétings are more or less official. But we would like to add a meeting that are only for a specified group or like a lokal internal traficday / weekend 
Is it some way to add a setting for that  kind of setting.
Like ", Only invited or local meeting"   that are only for a selected group or invited persons?

/Fredrik

**Comment by fjallemark** [comment]
There are two levels that this can be made:
- Meeting is public but only members in selected group can participate.
- Meeting is only visible for members in selected group. In order to see the meeting, a group member has to login first.

Advanced features:
- Send meeting invitation mails to group members.
- Check that meeting is only for members in FREMO groups in selected countries.
- Invite more that one group.

Things that are too complex:
- Invite individual persons. A workaround is to add temporary members to the invited group.

**Comment by fjallemark** [comment]
Internal meetings was added in release 1.6.15. Only group members in the arranging group can see and register to the meeting.
